src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionEditActionTest.java

```diff
@@ -1202,6 +1203,117 @@ public void testExecuteAndPostProcessRubricQuestion() throws Exception {
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWGIVERTO, FeedbackParticipantType.INSTRUCTORS.toString(),
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWRECIPIENTTO, FeedbackParticipantType.INSTRUCTORS.toString(),
+                Const.ParamsNames.FEEDBACK_QUESTION_EDITTYPE, "edit",
+                Const.ParamsNames.FEEDBACK_QUESTION_ID, fq.getId()
+        };
+
+        InstructorFeedbackQuestionEditAction a = getAction(params1);
+        a.executeAndPostProcess();
+
+        // Response rate should not change because other questions have the same respondents
+        fs = fsLogic.getFeedbackSession(fs.feedbackSessionName, fs.courseId);
+        details = fsLogic.getFeedbackSessionDetails(fs);
+        assertEquals(numStudentRespondents + numInstructorRespondents, details.stats.submittedTotal);
+        assertEquals(totalStudents + totalInstructors, details.stats.expectedTotal);
+
+        ______TS("Change the feedback path of a question with a unique instructor respondent");
+
+        fq = fqLogic.getFeedbackQuestion(fs.feedbackSessionName, fs.courseId, 3);
+        String[] params3 = {
+                Const.ParamsNames.COURSE_ID, fq.courseId,
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, fq.feedbackSessionName,
+                Const.ParamsNames.FEEDBACK_QUESTION_GIVERTYPE, fq.getGiverType().toString(),
+                Const.ParamsNames.FEEDBACK_QUESTION_RECIPIENTTYPE, FeedbackParticipantType.STUDENTS.toString(),
+                Const.ParamsNames.FEEDBACK_QUESTION_NUMBER, Integer.toString(fq.questionNumber),
+                Const.ParamsNames.FEEDBACK_QUESTION_TYPE, fq.getQuestionType().toString(),
+                Const.ParamsNames.FEEDBACK_QUESTION_TEXT, "My comments on the class",
+                Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIESTYPE, "1",
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWRESPONSESTO, FeedbackParticipantType.INSTRUCTORS.toString(),
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWGIVERTO, FeedbackParticipantType.INSTRUCTORS.toString(),
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWRECIPIENTTO, FeedbackParticipantType.INSTRUCTORS.toString(),
+                Const.ParamsNames.FEEDBACK_QUESTION_EDITTYPE, "edit",
+                Const.ParamsNames.FEEDBACK_QUESTION_ID, fq.getId()
+        };
+
+        a = getAction(params3);
+        a.executeAndPostProcess();
+
+        // Response rate should decrease by 1 because the response of the unique instructor respondent is deleted
+        fs = fsLogic.getFeedbackSession(fs.feedbackSessionName, fs.courseId);
+        details = fsLogic.getFeedbackSessionDetails(fs);
+        assertEquals(numStudentRespondents, details.stats.submittedTotal);
+        assertEquals(totalStudents + totalInstructors, details.stats.expectedTotal);
+
+        ______TS("Change the feedback path of a question so that some possible respondents are removed");
```

Comment:
Actually I feel a bit disturbed by these test, because now it will harder to add or modify existing questions in the typical data bundle, because we make these assumptions about the questions and responses here. I guess it's still fine though, since we won't expect much changes to the typical data bundle, but this is something to think about in future when you add tests.